config/environments/development.rb

```diff
@@ -72,12 +79,18 @@
   config.assets.quiet = true
 
   # Raises error for missing translations
-  # config.action_view.raise_on_missing_translations = true
+  config.i18n.raise_on_missing_translations = true
+
+  # Annotate rendered view with file names.
+  # config.action_view.annotate_rendered_view_with_filenames = true
```

Comment:
looks nice to enable in fact :)

Reply:
Yeah, I didn't look into it, but looks nice indeed.

Reply:
Well, turns out we can't enable it :grimacing: 

We have things like [this](https://github.com/3scale/porta/blob/39519eabe83a479530f1b7b04471e8af3a0cc5db/lib/developer_portal/app/views/developer_portal/admin/applications/keys/create.js.erb#L8) which uses this [render](https://github.com/3scale/porta/blob/39519eabe83a479530f1b7b04471e8af3a0cc5db/app/helpers/javascript_helper.rb#L4) and results in such JS:

```
    var keys = container.find("table#keys tbody");
    var key = "<!-- BEGIN app/views/provider/admin/keys/_key.html.erb --><tr id=\"application_key_9\" class=\"key\" data-key=\"27d572dd45df5966dee2de2ae5cbf642\">\n  <td>Application key<\/td>\n  <td class=\"key\" >27d572dd45df5966dee2de2ae5cbf642\n    <span class=\"delete_key operations\">\n        <a class=\"button-to action delete\" data-method=\"delete\" data-remote=\"true\" href=\"/p/admin/applications/7/keys/27d572dd45df5966dee2de2ae5cbf642\">Delete<\/a>\n<\/span>  <\/td>\n<\/tr><!-- END app/views/provider/admin/keys/_key.html.erb -->";
    $(key).hide().appendTo(keys.slice()).fadeIn();
```

:melting_face: 

So, I disabled it

Reply:
So is this an issue? I mean the HTML is valid. So whenever it is used, it should be valid. I guess not :)

Reply:
I guess it is, because jQuery was failing :shrug:

Reply:
Ok, some other day somebody may look into it